test/python/transpiler/test_scheduling_padding_pass.py

```diff
@@ -444,10 +444,18 @@ def test_measure_after_c_if_on_edge_locking(self):
 
         # lock at the end edge
         actual_asap = PassManager(
-            [ASAPSchedule(durations, clbit_write_latency=1000), PadDelay()]
+            [
+                SetIOLatency(clbit_write_latency=1000),
+                ASAPSchedule(durations),
+                PadDelay(),
+            ]
```

Comment:
Alternatively you could add `SetIOLatency` to the `self.requires` of `ASAPSchedule` and `ALAPSchedule` which guarantees that the i/o information is available to them.

Reply:
That sounds like a good plan. I'll keep this in mind.